src/widgets/auto-style/category-colors.js

```diff
@@ -15,12 +25,17 @@ var colorScales = [[
  *
  */
 
-function CategoryColors () {
+function CategoryColors (styles) {
+  this.updateColors(styles);
+}
+
+CategoryColors.prototype.updateColors = function updateColors (styles) {
```

Comment:
do we need a named function? or `CategoryColors.prototype.updateColors = function (styles) {` is enough?